eZ/Bundle/EzPublishCoreBundle/Resources/config/commands.yml

```
@@ -54,3 +54,11 @@ services:
             $ioService: '@ezpublish.fieldType.ezimage.io_service'
         tags:
             - { name: console.command }
+
+    Ibexa\Bundle\Core\Command\ExpireUserPasswordsCommand:
+        autowire: true
+        autoconfigure: true
+        arguments:
+            $userHandler: '@ezpublish.spi.persistence.legacy.user.handler'
```

This is Legacy storage handler implementation. What you want here is top layer handler rather:
```suggestion
            $userHandler: '@ezpublish.spi.persistence.user_handler'
```
unless... was there a specific reason for injecting that one?